### DISCUSSION TITLE
emmc or sd card with Rock Pi X

### DISCUSSION THREAD

**Discussion by bowseruk** [siderolabs/talos#5047] [Q&A]
I was looking to see if I could add a [Rock Pi X](https://wiki.radxa.com/RockpiX ) to a cluster to test Talos. I can get it to boot up from a USB stick with the AMD64 iso, but I can't get it to recognise the eMMC or SD Card. On Ubuntu both are working fine. I can't see any obvious kernel modules that are missing on the config compared to the Ubuntu one (looking at AMD64). I'm trying to figure out the best way to figure this out?

**Comment by smira** [comment]
This will be all about Linux kernel configuration. You might need to figure out the missing drivers by booting into Ubuntu and looking into the `/sys` tree, `lspsci` output, `dmesg`, etc. This is a tricky process.

Current config for Talos is https://github.com/talos-systems/pkgs/blob/master/kernel/build/config-amd64
